src/series.jl

```diff
@@ -21,9 +21,10 @@ abstract type AbstractRasterSeries{T,N,D,A} <: AbstractDimArray{T,N,D,A} end
 
 # Interface methods ####################################################
 
-DD.metadata(A::AbstractRasterSeries) = NoMetadata()
-DD.name(A::AbstractRasterSeries) = NoName()
+DD.metadata(A::AbstractRasterSeries) = DD.metadata(first(A))
+DD.name(A::AbstractRasterSeries) = DD.name(first(A))
```

Comment:
I don't know if this really is ever relevant. But after `combine`, the name/metadata of the first raster are preserved, so this at least is consistent.

Reply:
Its a bit weird... it makes sense after combine but not before it.